tests/ext/unit/test_aiohttp.py

```diff
@@ -706,3 +707,65 @@ def divide_by_zero(request):
         assert not context.hook_called
         await client.get("/")
         assert context.hook_called
+
+    async def test_unit__input_error__err__hook_called(self, aiohttp_client):
+        hapic = Hapic(async_=True, processor_class=MarshmallowProcessor)
+
+        class InputPathSchema(marshmallow.Schema):
+            user_id = marshmallow.fields.Int(required=True)
+
+        class MyContext(AiohttpContext):
+            def __init__(self, *args, **kwargs):
+                super().__init__(*args, **kwargs)
+                self.hook_called = False
+
+            def input_validation_error_caught(
+                self, hapic_data: HapicData, process_exception: ProcessException
+            ) -> None:
+                self.hook_called = True
+
+        @hapic.with_api_doc()
+        @hapic.input_path(InputPathSchema())
+        async def user():
+            pass
+
+        app = web.Application(debug=True)
+        context = MyContext(app)
+        hapic.set_context(context)
+        app.router.add_get("/user", user)
+        client = await aiohttp_client(app)
+
+        assert not context.hook_called
+        await client.get("/user")
+        assert context.hook_called
+
+    async def test_unit__output_error__err__hook_called(self, aiohttp_client):
```

Comment:
just a question, no typing here ?

Reply:
no typing in whole file so ... i didn't added it.

Reply:
ok